### ISSUE TITLE
Improve documentation and working example

### ISSUE THREAD

**Issue by jacodela** [CAMI-challenge/CAMISIM#42]
Hello,

I'm interested in simulating metagenome reads *de novo* using `CAMISIM`, however, I find the documentation hard to follow. The issues I have include:

* Links to obsolete pages: Most links to the *configuration file* either redirect to the top of the user manual or to [a page last updated in 2015](https://github.com/CAMI-challenge/CAMISIM/wiki/Configuration-File-Options).
* While `id_to_gff_file` is specified as required, there is no example of this file beyond a short explanation
* What is the difference between `genomes_total` and `genomes_real`?
* Is there a straightforward way of building a CAMI file without dealing with the hierarchical structure in the tax path and the percentage (e.g. only with a list of species taxIDs and their correspondent abundance)?

A working reproducible example and a guide to execute it is lacking, and would really facilitate the usage of `CAMISIM`

Thanks.

**Comment by AlphaSquad** [comment]
Thank you very much for your suggestions. I will have a look at the non-working links and deprecated pages in the manual.
Regarding your other questions:
 - The `id_to_gff_file` is not required anymore, `CAMISIM` should run with this option left empty. If this is not the case for you, it would be great if you could show us the example you used. I will change the corresponding part in the documentation.
 - `CAMISIM` is able to simulate strains of the input genomes using the sgEvolver from the mauve suite, if so desired. The number of artificially created strains is the difference between `genomes_total` and `genomes_real`. This is also what the `id_to_gff_file` is for: It maps genome IDs to predicted protein-coding sequences within the genomes which subsequently will not get evolved.
 - With "building a CAMI file" you mean the profiling output files? It is possible to use abundance files just like you describe as input for CAMI, these have to be specified using `distribution_file_paths` in the CommunityDesign section.

A `default_config` is available, but I agree that having a reproducible example might be warranted.

**Comment by AlphaSquad** [comment]
I already updated a lot of stuff in the [User manual](https://github.com/CAMI-challenge/CAMISIM/wiki/User-manual), I will work on a working minimal example soon.
Does that already help you?

**Comment by jacodela** [comment]
Thanks for your swift response! I will take a look to the manual and try to run it.
